src/pages/messaging/customer-chat-api/changelog/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
As far as I've noticed, a couple of links in the Customer API Changelog lead to `rtm-reference/`, for example **incoming greeting** https://deploy-preview-1035--livechat-public-docs.netlify.app/docs/messaging/customer-chat-api/changelog#greetings

Reply:
I have no idea how you found those, they seem to be the only ones left to change 😆 should be all good now